src/CMakeLists.txt

```diff
@@ -173,11 +179,11 @@ elseif(${CMAKE_CXX_COMPILER_ID} STREQUAL "GNU")
     # GCC's arry bounds, maybe uninitialized, and restrict warning seems to have false positives.
     target_compile_options(lexy_dev INTERFACE -Wno-array-bounds -Wno-maybe-uninitialized -Wno-restrict)
 elseif(MSVC)
-    target_compile_options(lexy_dev INTERFACE /WX /W3 /D _CRT_SECURE_NO_WARNINGS /wd5105)
+    target_compile_options(lexy_dev INTERFACE /WX /W3 /EHsc /D _CRT_SECURE_NO_WARNINGS /wd5105)
 endif()
 
-# Link to have FILE I/O.
-add_library(lexy_file)
+# Link to have FILE I/O. 
+add_library(lexy_file STATIC)
```

Comment:
Why hardcode STATIC? There is also trailing whitespace after the comment, please remove.

Reply:
Without it, on Windows you have to make the __declspec(dllexport) dance then, because the symbols are not exported by default. It will not produce any dll/lib, that causes problems later by the library clients:

> The following DLLs have no exports:
>
>   .../vcpkg/packages/lexy_x64-windows/debug/bin/lexy_file.dll
>    .../vcpkg/packages/lexy_x64-windows/bin/lexy_file.dll
>
> DLLs without any exports are likely a bug in the build script.

Reply:
Right, makes sense.